### PR TITLE
Feat/#56 create homescreen

### PR DESCRIPTION
**Pull request by emuguy1**
Added the basic layout stuff and created a new folder structure.
By pressing on the second icon you get to the homescreen.
Data has to be adjusted when entities are at that point.

### CODE REVIEW

File: app/src/main/java/de/othr/plantico/ui/PlantActivity.kt [hunk 1]
```diff
@@ -1,9 +1,10 @@
-package de.othr.plantico
+package de.othr.plantico.ui
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import androidx.activity.viewModels
-import androidx.lifecycle.asFlow
+import de.othr.plantico.TestViewModel
+import de.othr.plantico.TestViewModelFactory
```
Comment (tobireith) [inline app/src/main/java/de/othr/plantico/ui/PlantActivity.kt:7]: Are these imports used?

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -10,12 +11,17 @@
...
-            android:name=".PlantActivity"
+            android:name=".ui.PlantActivity"
             android:exported="false"
             android:label="@string/title_activity_plant"
             android:theme="@style/Theme.Plantico.NoActionBar" />
         <activity
-            android:name=".MainActivity"
+            android:name=".ui.homescreen.HomescreenActivity"
+            android:exported="false"
+            android:label="@string/app_name"
```
Comment (tobireith) [inline app/src/main/AndroidManifest.xml:21]: Should app_name be Plantico instead?

File: app/src/main/java/de/othr/plantico/ui/MainActivity.kt [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
+package de.othr.plantico.ui
 
+import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import androidx.activity.viewModels
 import androidx.recyclerview.widget.LinearLayoutManager
-import androidx.recyclerview.widget.RecyclerView
+import de.othr.plantico.TestViewModel
+import de.othr.plantico.TestViewModelFactory
```
Comment (tobireith) [inline app/src/main/java/de/othr/plantico/ui/MainActivity.kt:9]: Are these imports used?

**Review by tobireith — APPROVED** [review]

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (tobireith) [reply app/src/main/AndroidManifest.xml:21]: Okay nevermind, just saw it in the strings file.

File: app/src/main/java/de/othr/plantico/ui/MainActivity.kt (hunk 3 above)
Comment (emuguy1) [reply app/src/main/java/de/othr/plantico/ui/MainActivity.kt:9]: yes

**Pull request merged**